### ISSUE TITLE
Improve asking users to change network modal for MM wallets

### ISSUE THREAD

**Issue by creed-victor** [DistributedCollective/Sovryn-frontend#1027]
Metamask now has a feature which allows dapps to add new network and ask for network change  if user is in the wrong one:

```
window.ethereum.request({
  method: 'wallet_addEthereumChain',
  params: [{
    chainId: '0x38',
    chainName: 'Binance Smart Chain',
    nativeCurrency: { name: 'BNB', symbol: 'BNB', decimals: 18 },
    rpcUrls: ['https://bsc-dataseed.binance.org/'],
    blockExplorerUrls: ['https://bscscan.com/']
  }],
})
```

https://docs.metamask.io/guide/rpc-api.html#other-rpc-methods

https://consensys.net/blog/metamask/connect-users-to-layer-2-networks-with-the-metamask-custom-networks-api/
https://eips.ethereum.org/EIPS/eip-3085

**Comment by tiltom** [comment]
Regarding to the documentation (https://docs.metamask.io/guide/rpc-api.html#other-rpc-methods), this method should be called only after some user action (such as a button click). What is your proposed flow @t-z-u ?

**Comment by t-z-u** [comment]
I will have to look into this next week

**Comment by t-z-u** [comment]
https://xd.adobe.com/view/be4b50de-2b99-4506-5663-270ce738ee5f-c87a/

**Comment by tiltom** [comment]
On hold until the FISH Sale is not ready.

**Comment by tiltom** [comment]
Deployed.
